ui/push-health/PlatformConfig.jsx

```diff
@@ -88,11 +107,21 @@ class PlatformConfig extends React.PureComponent {
         style={{ background: '#f2f2f2' }}
       >
         <Row className="ml-2 w-100 mb-2 justify-content-between">
+          <Col xs="auto">
+            <Input
```

Comment:
Is it possible to make the check box a bit larger?  They're pretty small on my screen:
![Screen Shot 2020-09-28 at 3 32 02 PM](https://user-images.githubusercontent.com/419924/94492754-d3d03e80-019f-11eb-8aac-fa0eaf6569f1.png)


Maybe something along the lines of:

    width: 30px;
    height: 30px;
    margin-top: 0 !important;

Though that isn't quite vertically centered all the way.

Reply:
Achieved this using transform scale(1.2)